example/parse.js

```diff
@@ -1,4 +1,7 @@
 'use strict';
 
-var argv = require('../')(process.argv.slice(2));
+// const minimist = require('minimist'); // (normal include)
+const minimist = require('../'); // include minimist in git clone of minimist repo
```

Comment:
let's revert this, so the one-liner is in the example.

i also don't think we need the git clone version; if it's a concrete issue then we can add a package.json or something in the example folder

Reply:
Short version: Ok.

Long version: I run the examples from repos as a library maintainer to check the examples work, and check functionality. It might be little used by end users, especially since they files are so small. Using a "normal" import does make the example cleaner and match end-user code, so probably a win.

Reply:
it's totally fine to have a command that maintainers can run to test examples; the way to do that is to symlink the package into node_modules rather than altering the examples.

Reply:
Oh, interesting. Perhaps out of scope for Minimist but I am interested for Commander. Light question, do you have an example of that pattern somewhere in your repos?

Reply:
nah, i don't usually bother with it - but you can add `"minimist": "file:./"` to devDeps and it should work in most npm versions.

Reply:
I was experimenting with what is needed to import from `'minimist'`, and using `var argv = require('minimist')(process.argv.slice(2));` in the example file and running it directly from the repo seems to work fine. Finds the parent package by name just fine.  Can apparently use the "normal" include without any extra setup!

Reply:
If the package has `exports`, that will be true in node versions that support the `exports` field - since this package doesn't, i'm not sure why it would work :-/

Reply:
Urk. What on earth was I testing yesterday!? I will check again. (I have some theories...)

Reply:
Ok, I found this a bit amusing, despite my error. I was fooled by `node_modules` including a hoisted `minimist` due to a transient dependency.

```
% node --version
v20.11.1
% npm install
...
% ls -ld node_modules/minimist
drwxr-xr-x  12 john  staff  384 13 Mar 21:31 node_modules/minimist
% n 0.12.18
     copying : node/0.12.18
   installed : v0.12.18 (with npm 2.15.11)
% node --version
v0.12.18
% cat example/parse.js
'use strict';

var argv = require('minimist')(process.argv.slice(2));
console.log(argv);
% node example/parse.js
{ _: [] }
```